src/main/java/fi/fmi/avi/archiver/config/factory/postaction/SwimRabbitMQPublisherFactory.java

```diff
@@ -63,7 +75,18 @@ private static <T extends AutoCloseable> T createLazyForwardingProxy(
                             instance = instanceRef.get();
                         }
                     }
-                    return method.invoke(instance, args);
+                    try {
+                        return method.invoke(instance, args);
+                    } catch (final InvocationTargetException ite) {
+                        final Throwable cause = ite.getTargetException();
+                        if (cause instanceof RuntimeException re) {
```

Comment:
I'd suggest to let your IDE to search for variables that could be `final`. E.g. in `catch` or `instanceof` statements.

Reply:
Good comment, pushed me over the edge of laziness 👍